ios/Classes/models/Location.swift

```diff
@@ -6,11 +6,13 @@ public class Location : Codable
     let latitude: Double?
     let longitude: Double?
     let order: Int?
+    let isSilent: Bool
 
-    init(name: String, latitude: Double?, longitude: Double?, order: Int? = nil) {
+    init(name: String, latitude: Double?, longitude: Double?, order: Int? = nil, isSilent: Bool = false) {
         self.name = name
         self.latitude = latitude
         self.longitude = longitude
         self.order = order
+        self.isSilent = isSilent
```

Comment:
there is already an `isSilent` on `way_point.dart`. Can we use that instead of having it in 2 places?

Reply:
that's in swift not in Dart

Reply:
Ahh. Sorry